odl/util/utility.py

```diff
@@ -805,6 +805,43 @@ def pkg_supports(feature, pkg_version, pkg_feat_dict):
     return False
 
 
+class NumpyRandomSeed(object):
+    """Context manager for numpy randoms seeds."""
+
+    def __init__(self, seed):
+        """
```

Comment:
Why not? You can as well create an instance of the context manager and then write `with instance:`.